repos/system_upgrade/el7toel8/actors/checkhacluster/libraries/checkhacluster.py

```diff
@@ -25,6 +25,17 @@ def inhibit(node_type):
                 " to a RHEL High Availability or Resilient Storage Cluster"
             ),
         ),
+        reporting.Remediation(
+            hint=(
+                "Destroy the existing HA cluster"
```

Comment:
Hi @idevat , I might be wrong, but is there a way to check if the cluster is disabled from inside the actor? Then we could just update the condition to inhibit the upgrade (add the check of cluster state), rather than asking the user to remove the config files

Reply:
Red Hat does not support in-place upgrades or rolling-upgrades of cluster nodes from one major release of RHEL to another. Upgrade cannot continue if HA cluster exists.

Reply:
OK, then, is there a better way to check if HA cluster enabled (i.e. pcs cluster status) instead of a check of the existence of configs?

Reply:
Why you want only to disable HA cluster? The configuration from RHEL 7 will not properly work in RHEL 8. The HA cluster must be destroyed.

Reply:
Synced over chat. As I understood we can't do better and need to check configs files and if they exist, we have to ask the user to remove them as well.